src/components/AvatarSkeleton.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import {Circle} from 'react-native-svg';
+import themeColors from '@styles/themes/default';
+import SkeletonViewContentLoader from './SkeletonViewContentLoader';
+
+function AvatarSkeleton() {
```

Comment:
I'm not sure why we needed to create this. How do we display the loading avatar for report actions? Can't we reuse that?

Reply:
Sounds good.